Sources/Get/DataLoader.swift

```diff
@@ -357,7 +357,7 @@ func encode(_ value: Encodable, using encoder: JSONEncoder) async throws -> Data
         return string.data(using: .utf8)
     } else {
         return try await Task.detached {
-            try encoder.encode(AnyEncodable(value: value))
+            try encoder.encode(value)
```

Comment:
I think `AnyEncodable` can now also be removed. It was probably a rudiment from one of the earlier implementations – I can't remember now.

Reply:
there might have been a time when AnyEncodable didn't support Optionals or something

Reply:
want me to remove and update this PR? or you want to do it?

Reply:
I would appreciate if you could remove it, but I can do it myself later.

Reply:
I removed it in main